doc/DDT.md

```diff
@@ -43,3 +43,5 @@ The ◊ character represents typing <kbd>ESC</kbd>, or holding down the
 | man 2 open    |                      | :call open               |
 | dmesg         |                      | :sysmsg                  |
 | uptime        |                      | :time                    |
+| echo "Hello World!" | | <kbd>^W</kbd>:◊<kbd>^V</kbd>Hello World! <kbd>^W</kbd>◊<kbd>^V</kbd> *(in an xfile)* |
+| \#Comment     |       | <kbd>^W</kbd>:◊Comment◊<kbd>^V</kbd> *(in an xfile)* |
```

Comment:
You have the comment inside a ^W...^V pair which makes the comment invisible in the output.  From reading xfile scripts, I think it's actually more idiomatic to leave the comments visible in the output.  On the other hand, hiding the comment is closer to the Unix behavior.